packages/components/src/checkbox/checkbox.test.tsx

```diff
@@ -535,11 +535,3 @@ test("On resetting form, all checkboxes in the form should reset to its default
   expect(checkbox1).toBeChecked()
   expect(checkbox2).not.toBeChecked()
 })
-
-test("Checkbox with an ID should have a for on the label", () => {
```

Comment:
@segunadebayo It seems this `for` attribute was removed here https://github.com/chakra-ui/chakra-ui/commit/34802a2e1ae17daf35441fd4f2da9cbe82094a41

Reply:
Good catch! I've added it back